src/index.js

```diff
@@ -53,11 +55,13 @@ class DropdownTreeSelect extends Component {
       showDropdown: this.props.showDropdown || false,
       searchModeOn: false
     }
+    // This needs id generator from https://github.com/dowjones/react-dropdown-tree-select/pull/220
+    this.clientId = this.props.id
```

Comment:
Can we take out `id` related changes? Those would come from 220 anyway and would simplify the diff ehre.

Reply:
What do you mean, the comment or the line itself? It's needed for the rootPrefixId (inner ids)

Reply:
No, I mean `this.clientId = this.props.id` and anything related to id generation. If you need id changes here, then let's try and merge that PR first.

Reply:
I need the id for a shared name on radios. It still depends om the other pr.